### PR TITLE
Update Deep learning research for EHR

### PR DESCRIPTION
**Pull request by hurcy**
This is for variety deep learning techniques and frameworks applied to clinical applications.

https://github.com/hurcy/awesome-ehr-deeplearning

Over the past decade, the volume of electronic health records has exploded. This data has great potential. Thanks to advances in machine learning and deep learning techniques, health records have been converted into mathematical expressions. We provide a collection of must-read papers on various EHR topics - recent research trends, applications to predict patient outcomes, visualization of complex data.

### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

## Requirements for your pull request

- I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title.<br>For example, `Add Name of List`, not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should:
	- Include a short description about the project/theme of the list. **It should not describe the list itself.**<br>Example: `- [Fish](…) - User-friendly shell.`, not `- [Fish](…) - Resources for Fish.`.
	- Be added at the bottom of the appropriate category.
- The list I'm submitting complies with these requirements:

## Requirements for your Awesome list

- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
- It's the result of hard work and the best I could possibly produce.
- Non-generated Markdown file in a GitHub repo.
- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
- Not a duplicate.
- Only has awesome items. Awesome lists are curations of the best, not everything.
- Includes a project logo/illustration whenever possible.
	- Either fullwidth or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
	- The image should link to the project website or any relevant website.
	- The image should be high-DPI. Set it to maximum half the width of the original image.
- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
- Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge).
	- Should be placed on the right side of the readme heading.
	- Should link back to this list.
- Has a Table of Contents section.
	- Should be named `Contents`, not `Table of Contents`.
	- Should be the first section in the list.
	- Should only have one level of sub-lists, preferably none.
- Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).
	- That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**
	- [WTFPL](http://www.wtfpl.net) and [Unlicense](http://unlicense.org) are not acceptable licenses.
	- If you use a license badge, it should be SVG, not PNG.
- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
	- The file should be named `contributing.md`. Casing is up to you.
- Has consistent formatting and proper spelling/grammar.
	- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
	- The description starts with an uppercase character and ends with a period.
	- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.

Go to the top and read it again.

### CODE REVIEW

**Comment by legodude17** [comment]
Please add the URL at the top of the PR description. Also remove the frontmatter table in the readme of the list. It would be nice if it was longer, and descriptions would be very helpful.

**Comment by hurcy** [comment]
@legodude17 Thanks. I updated.

**Comment by sindresorhus** [comment]
Hey, sorry for the lack of response, please see #1363.

**Comment by sindresorhus** [comment]
@hurcy Is this ready to be reviewed again?

**Comment by ianchanning** [comment]
@hurcy your links should be formatted as per the requirements:

> The link and description are separated by a dash.
> `Example: - [AVA](…) - JavaScript test runner.`

Currently you have:

> `**Paper title** authors [pdf](...)`

I think it would be also useful to specify the year for each paper.

**Comment by hurcy** [comment]
@ianchanning Updated! Also papers are sorted by date published.

**Comment by sindresorhus** [comment]
Can you fix the merge conflict?

**Comment by sindresorhus** [comment]
I don't see the point of `[pdf] - `. Just make the title the link to the PDF. The formatting should be `- [Title](URL) - Description.`

**Comment by sindresorhus** [comment]
It would also be useful to include a short summary for each item.

**Comment by sindresorhus** [comment]
```
❯ awesome-lint https://github.com/hurcy/awesome-ehr-deeplearning
✖ Linting

  README.md:104:4
  ✖    1:1  Missing License section                      remark-lint:awesome/license
  ✖   11:1  Multiple Table of Contents lists found       remark-lint:awesome/toc
  ✖   11:1  Table of Contents must be the first section  remark-lint:awesome/toc
  ✖   19:1  Rules should use ---                         remark-lint:rule-style
  ✖  104:4  Incorrect list-item indent: remove 1 space   remark-lint:list-item-indent
  ✖  114:1  Rules should use ---                         remark-lint:rule-style

  6 errors
```

https://github.com/sindresorhus/awesome-lint

File: readme.md [hunk 1]
```diff
@@ -598,7 +598,7 @@
 - [Jupyter](https://github.com/markusschanta/awesome-jupyter#readme) - Create and share documents that contain code, equations, visualizations and narrative text.
 - [FIRST Robotics Competition](https://github.com/andrewda/awesome-frc#readme) - International high school robotics championship.
 - [Humane Technology](https://github.com/engagingspaces/awesome-humane-tech#readme) - Open source projects that help improve society.
-
+- [Electronics Health Records](https://github.c/awesome-ehr-deeplearning) - Deep Learning and EHRs.
```
Comment (sindresorhus) [inline readme.md]: The title here should be the same as your list heading and the description should succinctly explain what EHR and deep learning is.

**Comment by sindresorhus** [comment]
> * **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)

**Comment by sindresorhus** [comment]
The repo description is out of sync with the readme.

**Comment by sindresorhus** [comment]
Closing for lack of response.

**Pull request closed without merging**